lib/toml-parser.js

```diff
@@ -115,9 +115,13 @@ const _type = Symbol('type')
 const _declared = Symbol('declared')
 
 const hasOwnProperty = Object.prototype.hasOwnProperty
+const defineProperty = Object.defineProperty
+const descriptor = Object.assign(Object.create(null), {configurable: true, enumerable: true, writable: true, value: void 0})
```

Comment:
`undefined` is preferred to `void 0` (strict mode makes this 100% safe)

Reply:
I am trying to figure out why on earth you're doing the Object.assign/Object.create dance here and I can see no reason? Can you explain why the an argument to defineProperty benefits from being a prototypeless object?

Reply:
@iarna Just to avoid Object.prototype was defined `set` or `get` property, and that will make `defineProperty` throw an error. Maybe it's your repo, not my own, so I want to make sure my PR is 100% safe to you in any case. I cancel it now, by using the normal way.

Reply:
Ahh, yeah, guarding against monkey patching. Gotcha. I tend to avoid those kinds of patterns, because I'll admit I treat monkey patching in terms of "you break it, you get to keep the pieces" but I can see how that wouldn't universally be true.